CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
   * enable/disable metrics in the KafkaBridge custom resource
   * new Grafana dashboard for the bridge metrics
 * Support dynamically changeable logging in the Entity Operator and Kafka Bridge 
+* Configure Cluster Operator logging using ConfigMap instead of environment variable and support dynamic changes
```

Comment:
FYI: Looks like this will not get into 0.19.0, so you might need to move this to 0.20.0.

Reply:
@stanlyDoge I still see this here in 0.19.0, should you remove it as asked by Jakub?